deepchem/metrics/__init__.py

```diff
@@ -4,6 +4,7 @@
 import warnings
 import sklearn.metrics
 import logging
+# TODO: Imported metrics will be removed in a futrue version of DeepCHem
```

Comment:
Do we really want to do that?  I don't think it does any harm to import them here, and it's easier to be able to get all metrics from `dc.metrics` instead of having to remember which ones are there and which ones are in sklearn.

Reply:
Good point. I think you're right. I'll remove these deprecations and add them to the docs.